### ISSUE TITLE
Is there an option to disable certificate validation when sending SMTP

### ISSUE THREAD

**Issue by guidoffm** [jdrouet/catapulte#173]
I am getting 

`{"name":"Internal Server Error","message":"internal client error: timed out waiting for connection: network error: invalid certificate: UnknownIssuer"}`

since the certificates ca certificate in unknown in the container.

**Comment by guidoffm** [comment]
You have a comment here:

https://github.com/jdrouet/catapulte/blob/main/src/service/smtp.rs#L75

**Comment by jdrouet** [comment]
Not yet, but if you're motivated enough, fill free to open a PR 😉 
Otherwise, I'll probably take a look at that this weekend or next week.

**Comment by guidoffm** [comment]
I have tried other options to get it running with TLS SMTP:

- I have created another Docker file based on Debian, added the root certificates of our SMTP relay host. No luck.
- I switched to unencrypted SMTP on port 25. Still no luck. The program complained about no available authentication methods. Smart host was Office 365 for that.

The only working solution I found for me was to run a second container in parallel with the image https://hub.docker.com/r/namshi/smtp/
